sdcm/cluster.py

```diff
@@ -3353,13 +3352,16 @@ def execute_cmd(cql_session, entity_type):
             else:
                 raise ValueError(f"The following value '{entity_type}' not supported")
 
-            session.default_fetch_size = 1000
-            session.default_consistency_level = ConsistencyLevel.ONE
-            execute_result = session.execute_async(cmd)
-            fetcher = PageFetcher(execute_result).request_all()
-            current_rows = fetcher.all_data()
```

Comment:
what is exactly the problem with `PageFetcher` ? 
in you code you are not doing pagination at all, and you might get partial data